src/Router/SimpleRouter.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Equip\Queue\Router;
+
+use Equip\Queue\Exception\HandlerException;
+use Equip\Queue\Exception\RouterException;
+
+class SimpleRouter implements RouterInterface
+{
+    /**
+     * @var array
+     */
+    private $routes;
+
+    /**
+     * @param array $routes
+     */
+    public function __construct(array $routes = [])
+    {
+        $this->routes = $routes;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function get($handler)
+    {
+        $route = $this->getRoute($handler);
+        if (!$route) {
+            throw RouterException::routeNotFound($handler);
+        }
+
+        if (is_string($route) && class_exists($route)) {
+            $route = new $route;
```

Comment:
This is a simple implementation, the idea is that if users need more they can just implement `RouterInterface`.